src/code42cli/cmds/securitydata.py

```diff
@@ -206,12 +206,12 @@ def clear_checkpoint(state, checkpoint_name):
 def _call_extractor(
     state, handlers, begin, end, or_query, advanced_query, saved_search, **kwargs
 ):
+    if advanced_query:
+        state.search_filters = advanced_query
     extractor = _get_file_event_extractor(state.sdk, handlers)
     extractor.use_or_query = or_query
     extractor.or_query_exempt_filters.append(f.ExposureType.exists())
-    if advanced_query:
-        extractor.extract_advanced(advanced_query)
-    elif saved_search:
+    if saved_search:
```

Comment:
use `elif` here.

Reply:
Nope, the `saved-search` uses `cls=incompatible_with("advanced_query")`.